src/Playlist.js

```diff
@@ -645,7 +686,7 @@ export default class {
         let activeTrack = this.getActiveTrack();
         let trackElements = this.tracks.map((track) => {
             return track.render(this.getTrackRenderData({
-                "isActive": (activeTrack === track) ? true : false,
+                "isActive": (this.getTimeSelection().start!=this.getTimeSelection().end)?((activeTrack === track) ? true : false):true,
```

Comment:
I have not realized that i wrote two time the same line. It could be interesting to create a new function for that. ( we are talking about `this.getTimeSelection.start!=...`, isn't it ? )